lesson7-questions.txt

```diff
@@ -67,14 +78,17 @@ Try that last line.  You will see that it creates a subscription object, but tha
 object is not entirely empty.  Which attribute is set? Where does it get that
 value?
 
+A subscription object is returned. The attribute that is set is the user_id attribute. Its automatically
```

Comment:
A subscription object would have methods like `forum_id`, `user_id`, and `priority`. However, we cannot do this in the Rails console:

```ruby
user=User.first
subscriptions = user.subscriptions
puts user.subscriptions.forum_id # <-- raises a NoMethodError
```

What's really returned?